tests/runs/test_cli.py

```diff
@@ -55,6 +74,38 @@ def test_submit_cli_json(runs_api_mock):
             SUBMIT_JSON)
         assert echo_mock.call_args[0][0] == pretty_format(SUBMIT_RETURN)
 
+@provide_conf
+def test_submit_wait_success(runs_api_mock):
+    runs_api_mock.submit_run.return_value = SUBMIT_RETURN
+    runs_api_mock.get_run.return_value = RUNS_GET_RETURN_SUCCESS
+    runner = CliRunner()
+    result = runner.invoke(cli.submit_cli, ['--json', SUBMIT_JSON, '--wait'])
+    runs_api_mock.get_run.assert_called_once()
+    assert result.exit_code == 0
+
+@provide_conf
+def test_submit_wait_failure(runs_api_mock):
```

Comment:
Should we also add a test for eventual failure, and for other terminal states besides `TERMINATED` (i.e. test the `INTERNAL_ERROR` and `SKIPPED` life_cycle_state cases)? You can use the `pytest.mark.parametrize` API to parametrize your test across different terminal states ([see docs](https://docs.pytest.org/en/6.2.x/parametrize.html)).

Reply:
Done